sass/_Omnibox.scss

```diff
@@ -5,18 +5,35 @@
   z-index: 0;
 }
 .CoveoOmnibox.magic-box {
-  .coveo-omnibox-hightlight {
+  .coveo-omnibox-suggestion-input {
     font-weight: bold;
     color: $black;
   }
   .magic-box-suggestions {
     z-index: 3;
   }
-  .coveo-omnibox-hightlight2 {
-    font-weight: bold;
+  .coveo-omnibox-suggestion-input-corrected {
     font-style: italic;
     color: $black;
   }
+  .magic-box-suggestion {
+    font-weight: bold;
+    color: $black;
+
+    .coveo-omnibox-suggestion-input {
+      font-weight: initial;
+    }
+
+    .coveo-omnibox-suggestion-input-corrected {
+      font-weight: initial;
+    }
+
+    .coveo-omnibox-suggestion-category,
+    .coveo-omnibox-suggestion-results-count {
+      font-weight: initial;
+      opacity: 0.4;
```

Comment:
Like I mentioned in the PR, 0.4 could be a bit much for contrast issues.  It would also be better to use a `color` variable rather than  `opacity`

Reply:
Alright, changed to use `color` instead and tested it with the Axe extension.